src/ur5e_arm.cpp

```diff
@@ -219,7 +219,22 @@ void UR5eArm::stop(const ProtoStruct& extra) {
 }
 
 ProtoStruct UR5eArm::do_command(const ProtoStruct& command) {
-    return ProtoStruct{};
+    ProtoStruct resp = ProtoStruct{};
+
+    for (auto kv : command) {
+        if (kv.first == VEL_KEY) {
+            const double val = *kv.second.get<double>();
+            speed.store(val * (M_PI / 180.0));
+            resp.emplace(VEL_KEY, val);
+        }
+        if (kv.first == ACC_KEY) {
+            const double val = *kv.second.get<double>();
+            acceleration.store(val * (M_PI / 180.0));
+            resp.emplace(ACC_KEY, val);
+        }
+    }
+
+    return resp;
```

Comment:
Why return a response populated with the things you passed in?  I would expect this to return an empty struct similar to how it was before. Is there some RDK idiom with DoCommands I'm not aware of?